hack/pr-deploy-prpqr-ui.yaml

```
@@ -103,7 +103,7 @@ objects:
         name: "output:binaries"
     source:
       dockerfile: |
-        FROM registry.ci.openshift.org/openshift/release:golang-1.19
+        FROM registry.ci.openshift.org/ci/origin-release:golang-1.21
```

```suggestion
        FROM registry.ci.openshift.org/openshift/release:golang-1.21
```

You could change only the tags.

```console
$ oc get istag -n openshift release:golang-1.21
NAME                  IMAGE REFERENCE                                                                                                                              UPDATED
release:golang-1.21   image-registry.openshift-image-registry.svc:5000/openshift/release@sha256:62898d6f93588e517e07749e323063b7432d550fb2e311f57c4a64d6d3012108   13 hours ago
```

I would use the build-root image instead as we discussed to keep it consistent with the build itself.
But that is not very in the scope of this PR tho.